docs/_ext/paramtable.py

```diff
@@ -55,17 +54,14 @@ def run(self):
                     entry += para
                 elif c == "name":
                     text = d[c]
-                    alias_list = d.get("aliases", [])
-                    if alias_list:
+                    if alias_list := d.get("aliases", []):
```

Comment:
I guess as long as our min version is 3.8 this operator is probably OK, but I don't think we use it widely elsewhere.

Reply:
I think this is a useful shorthand.

Since oldestdeps tests are succeeding, I don't think this will create any issue.